unity-client/Assets/Scripts/MainScripts/DCL/Helpers/UIHelpers/DynamicScrollSensitivity.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Let to set a dynamic sensitivity value depending on the height of the content container.
+/// Call to RecalculateSensitivity() function to calculate and apply the new sensitivity value.
+/// </summary>
+public class DynamicScrollSensitivity : MonoBehaviour
+{
+    [Tooltip("Scroll Rect component that will be modified.")]
+    public ScrollRect mainScrollRect;
+    [Tooltip("Viewport associated to the Scroll Rect.")]
+    public RectTransform viewport;
+    [Tooltip("Transform that will contain all the items of the viewport.")]
+    public RectTransform contentContainer;
+    [Tooltip("Min value for the calculated scroll sensitivity.")]
+    public float minSensitivity = 10f;
+    [Tooltip("Max value for the calculated scroll sensitivity.")]
+    public float maxSensitivity = 100f;
+    [Tooltip("True to recalculate each time the game object is enabled.")]
+    public bool recalculateOnEnable = true;
+
+    private Coroutine recalculateCoroutine = null;
+
+    private void OnEnable()
+    {
+        if (recalculateOnEnable)
+            RecalculateSensitivity();
+    }
+
+    /// <summary>
+    /// Recalculate the scroll sensitivity value depending on the current height of the content container.
+    /// </summary>
+    [ContextMenu("Recalculate Sensitivity")]
+    public void RecalculateSensitivity()
```

Comment:
I like the context menus but I think that we are not including them in master is that correct @AjimenezDCL ?

Reply:
We try not to include testing code in our main repository and I think this is an example for that (nice catch!). Nevertheless, we do have some cases where `ContextMenu` are handy, like tools and editor features.

Reply:
Ups, yeah, I used it for testing purposes and I forgot to remove it. I will remove it, thanks!